lms/lib/comment_client/comment.py

```diff
@@ -30,6 +30,11 @@ class Comment(models.Model):
     def thread(self):
         return Thread(id=self.thread_id, type='thread')
 
+    @property
+    def context(self):
```

Comment:
You don't use this property now that you are doing getattr, correct? Any reason to keep this code?